### PR TITLE
document: simulate file fault in physical machine by using chaosd

### LINKED ISSUE #249: add document for injecting fault to file for chaosd
related pr https://github.com/chaos-mesh/chaosd/pull/123

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
close #249 

TODO: translate to English

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md [hunk 1]
```diff
@@ -0,0 +1,440 @@
...
+      --uid string         the experiment ID
+```
+
+#### 写文件相关配置说明
+
+| 配置项 | 配置缩写 | 说明 | 值 |
+| :-- | :-- | :-- | :-- |
+| `count` | c | 写数据的次数 | Int，默认值为 1 |
+| `data` | d | 要写入文件的数据 | String，例如 "test"，必须要设置 |
+| `file-name` | f | 要写入数据的文件路径 | String，例如 "/tmp/test.txt"，必须要设置 |
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md:80]:
```suggestion
| `count` | c | 写数据的次数 | int，默认值为 `1` |
| `data` | d | 要写入文件的数据 | string，例如 `"test"`，必须要设置 |
| `file-name` | f | 要写入数据的文件路径 | string，例如 `"/tmp/test.txt"`，必须要设置 |
```

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md [hunk 2]
```diff
@@ -0,0 +1,440 @@
...
+      --log-level string   the log level of chaosd. The value can be 'debug', 'info', 'warn' and 'error'
+      --uid string         the experiment ID
+```
+
+#### 创建文件相关配置说明
+
+| 配置项 | 配置缩写 | 说明 | 值 |
+| :-- | :-- | :-- | :-- |
+| `dir-name` | d | 创建的目录名称 | String，例如 "/tmp/test"，`dir-name` 和 `file-name` 必须要设置其中一个 |
+| `file-name` | f | 创建的文件名称 | String，例如 "/tmp/test.txt"，`dir-name` 和 `file-name` 必须要设置其中一个 |
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md:123]:
```suggestion
| `dir-name` | d | 创建的目录名称 | string，例如 `"/tmp/test"`，`dir-name` 和 `file-name` 必须要设置其中一个 |
| `file-name` | f | 创建的文件名称 | string，例如 `"/tmp/test.txt"`，`dir-name` 和 `file-name` 必须要设置其中一个 |
```

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md [hunk 3]
```diff
@@ -0,0 +1,440 @@
...
+      --log-level string   the log level of chaosd. The value can be 'debug', 'info', 'warn' and 'error'
+      --uid string         the experiment ID
+```
+
+#### 删除文件相关配置说明
+
+| 配置项 | 配置缩写 | 说明 | 值 |
+| :-- | :-- | :-- | :-- |
+| `dir-name` | d | 删除的目录名称 | String，例如 "/tmp/test"，`dir-name` 和 `file-name` 必须要设置其中一个 |
+| `file-name` | f | 删除的文件名称 | String，例如 "/tmp/test.txt"，`dir-name` 和 `file-name` 必须要设置其中一个 |
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md:166]:
```suggestion
| `dir-name` | d | 删除的目录名称 | string，例如 `"/tmp/test"`，`dir-name` 和 `file-name` 必须要设置其中一个 |
| `file-name` | f | 删除的文件名称 | string，例如 `"/tmp/test.txt"`，`dir-name` 和 `file-name` 必须要设置其中一个 |
```

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md [hunk 4]
```diff
@@ -0,0 +1,440 @@
...
+      --log-level string   the log level of chaosd. The value can be 'debug', 'info', 'warn' and 'error'
+      --uid string         the experiment ID
+```
+
+#### 修改文件权限相关配置说明
+
+| 配置项 | 配置缩写 | 说明 | 值 |
+| :-- | :-- | :-- | :-- |
+| `file-name` | f | 要修改权限的文件名称 | String，例如 "/tmp/test.txt"，必须要设置 |
+| `privilege` | p | 将文件权限修改为该值 | Int，例如 777，必须要设置 |
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md:209]:
```suggestion
| `file-name` | f | 要修改权限的文件名称 | string，例如 `"/tmp/test.txt"`，必须要设置 |
| `privilege` | p | 将文件权限修改为该值 | int，例如 `777`，必须要设置 |
```

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md [hunk 5]
```diff
@@ -0,0 +1,440 @@
...
+      --log-level string   the log level of chaosd. The value can be 'debug', 'info', 'warn' and 'error'
+      --uid string         the experiment ID
+```
+
+#### 重命名文件相关配置说明
+
+| 配置项 | 配置缩写 | 说明 | 值 |
+| :-- | :-- | :-- | :-- |
+| `dest-file` | d | 将文件重命名为该值 | String，例如 "/tmp/test2.txt"，必须要设置 |
+| `source-file` | s | 要重命名的文件名称 | String，例如 "/tmp/test.txt"，必须要设置 |
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md:252]:
```suggestion
| `dest-file` | d | 目标文件名称 | string，例如 `"/tmp/test2.txt"`，必须要设置 |
| `source-file` | s | 源文件名称 | string，例如 `"/tmp/test.txt"`，必须要设置 |
```

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md [hunk 6]
```diff
@@ -0,0 +1,440 @@
...
+```
+
+#### 替换文件内容相关配置说明
+
+| 配置项 | 配置缩写 | 说明 | 值 |
+| :-- | :-- | :-- | :-- |
+| `dest-string` | d | 将文件中的内容替换为该值 | String，例如 "text"，必须要设置 |
+| `file-name` | f | 要替换内容的文件名称 | String，例如 "/tmp/test.txt"，必须要设置 |
+| `line` | l | 替换文件中哪一行的数据 | Int，默认为 0，表示替换所有能匹配到 `origin-string` 的行的数据 |
+| `origin-string` | o | 文件中要替换的数据 | String，例如 "test"，必须要设置 |
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md:299]:
```suggestion
| `dest-string` | d | 将文件中的内容替换为该值 | string，例如 `"text"`，必须要设置 |
| `file-name` | f | 要替换内容的文件名称 | string，例如 `"/tmp/test.txt"`，必须要设置 |
| `line` | l | 替换文件中哪一行的数据 | int，默认为 `0`，表示替换所有能匹配到 `origin-string` 的行的数据 |
| `origin-string` | o | 文件中要替换的数据 | string，例如 `"test"`，必须要设置 |
```

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md [hunk 7]
```diff
@@ -0,0 +1,440 @@
...
+
+    ```bash
+    curl -X POST 172.16.112.130:31767/api/attack/jvm -H "Content-Type:application/json" -d '{fault-configuration}'
+    ```
+
+    在上述命令中，你需要按照故障类型在 `fault-configuration` 中进行配置。有关对应的配置参数，请参考下文中各个类型故障的相关参数说明和命令示例。
+
+:::note 注意
+
+在运行实验时，请注意保存实验的 UID 信息。当要结束 UID 对应的实验时，需要向 Chaosd 服务的路径 /api/attack/{uid} 发送 `DELETE` HTTP 请求。
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md:327]:
```suggestion
在运行实验时，请注意保存实验的 UID 信息。当要结束 UID 对应的实验时，需要向 Chaosd 服务的路径 `/api/attack/{uid}` 发送 `DELETE` HTTP 请求。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md [hunk 8]
```diff
@@ -0,0 +1,440 @@
...
+通过该功能以追加的方式将数据写到文件的末尾。
+
+#### 写文件相关参数说明
+
+| 参数 | 说明 | 值 |
+| :-- | :-- | :-- |
+| `action` | 实验的行为 | 设置为 `"append"` |
+| `count` | 写数据的次数 | Int，默认值为 1 |
+| `data` | 要写入文件的数据 | String，例如 "test"，必须要设置 |
+| `file-name` | 要写入数据的文件路径 | String，例如 "/tmp/test.txt"，必须要设置 |
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md:342]:
```suggestion
| `count` | 写数据的次数 | int，默认值为 `1` |
| `data` | 要写入文件的数据 | string，例如 `"test"`，必须要设置 |
| `file-name` | 要写入数据的文件路径 | string，例如 `"/tmp/test.txt"`，必须要设置 |
```

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md [hunk 9]
```diff
@@ -0,0 +1,440 @@
...
+
+通过该功能可以创建新的文件或者目录。
+
+#### 创建文件相关参数说明
+
+| 参数 | 说明 | 值 |
+| :-- | :-- | :-- |
+| `action` | 实验的行为 | 设置为 `"create"` |
+| `dir-name` | 创建的目录名称 | String，例如 "/tmp/test"，`dir-name` 和 `file-name` 必须要设置其中一个 |
+| `file-name` | 创建的文件名称 | String，例如 "/tmp/test.txt"，`dir-name` 和 `file-name` 必须要设置其中一个 |
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md:360]:
```suggestion
| `dir-name` | 创建的目录名称 | string，例如 `"/tmp/test"`，`dir-name` 和 `file-name` 必须要设置其中一个 |
| `file-name` | 创建的文件名称 | string，例如 `"/tmp/test.txt"`，`dir-name` 和 `file-name` 必须要设置其中一个 |
```

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md [hunk 10]
```diff
@@ -0,0 +1,440 @@
...
+
+使用该功能修改文件的权限。
+
+#### 修改文件权限相关参数说明
+
+| 参数 | 说明 | 值 |
+| :-- | :-- | :-- |
+| `action` | 实验的行为 | 设置为 `"modify"` |
+| `file-name` | 要修改权限的文件名称 | String，例如 "/tmp/test.txt"，必须要设置 |
+| `privilege` | 将文件权限修改为该值 | Int，例如 777，必须要设置 |
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md:396]:
```suggestion
| `file-name` | 要修改权限的文件名称 | string，例如 `"/tmp/test.txt"`，必须要设置 |
| `privilege` | 将文件权限修改为该值 | int，例如 `777`，必须要设置 |
```

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md [hunk 11]
```diff
@@ -0,0 +1,440 @@
...
+
+使用该功能重命名文件。
+
+#### 重命名文件相关参数说明
+
+| 参数 | 说明 | 值 |
+| :-- | :-- | :-- |
+| `action` | 实验的行为 | 设置为 `"rename"` |
+| `dest-file` | 将文件重命名为该值 | String，例如 "/tmp/test2.txt"，必须要设置 |
+| `source-file` | 要重命名的文件名称 | String，例如 "/tmp/test.txt"，必须要设置 |
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md:414]:
```suggestion
| `dest-file` | 将文件重命名为该值 | string，例如 `"/tmp/test2.txt"`，必须要设置 |
| `source-file` | 要重命名的文件名称 | string，例如 `"/tmp/test.txt"`，必须要设置 |
```

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md [hunk 12]
```diff
@@ -0,0 +1,440 @@
...
+
+#### 替换文件内容相关参数说明
+
+| 参数 | 说明 | 值 |
+| :-- | :-- | :-- |
+| `action` | 实验的行为 | 设置为 `"replace"` |
+| `dest-string` | 将文件中的内容替换为该值 | String，例如 "text"，必须要设置 |
+| `file-name` | 要替换内容的文件名称 | String，例如 "/tmp/test.txt"，必须要设置 |
+| `line` | 替换文件中哪一行的数据 | Int，默认为 0，表示替换所有能匹配到 `origin-string` 的行的数据 |
+| `origin-string` | 文件中要替换的数据 | String，例如 "test"，必须要设置 |
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md:434]:
```suggestion
| `dest-string` | 将文件中的内容替换为该值 | string，例如 `"text"`，必须要设置 |
| `file-name` | 要替换内容的文件名称 | string，例如 `"/tmp/test.txt"`，必须要设置 |
| `line` | 替换文件中哪一行的数据 | int，默认为 `0`，表示替换所有能匹配到 `origin-string` 的行的数据 |
| `origin-string` | 文件中要替换的数据 | string，例如 `"test"`，必须要设置 |
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Comment by WangXiangUSTC** [comment]
@Oreoxmt PTAL again

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md [hunk 13]
```diff
@@ -0,0 +1,440 @@
...
+```
+
+#### 替换文件内容相关配置说明
+
+| 配置项 | 配置缩写 | 说明 | 值 |
+| :-- | :-- | :-- | :-- |
+| `dest-string` | d | 将文件中的内容替换为该值 | string，例如 `"text"`，必须要设置 |
+| `file-name` | f | 要替换内容的文件名称 | string，例如 `"/tmp/test.txt"`，必须要设置 |
+| `line` | l | 替换文件中哪一行的数据 | int，默认为 `0`，表示替换所有能匹配到 `origin-string` 的行的数据 |
+| `origin-string` | o | 文件中要替换的数据 | String，例如 "test"，必须要设置 |
```
Comment (Oreoxmt) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md:299]:
```suggestion
| `origin-string` | o | 文件中要替换的数据 | string，例如 `"test"`，必须要设置 |
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md (hunk 13 above)
Comment (WangXiangUSTC) [reply i18n/zh/docusaurus-plugin-content-docs/current/simulate-file-chaos-in-physical-nodes.md]: done

**Review by Oreoxmt — APPROVED** [review]

**Review by cwen0 — APPROVED** [review]
LGTM

**Pull request merged**